### DISCUSSION TITLE
Add support for non-populated submodules in `magit-list-submodules`

### DISCUSSION THREAD

**Discussion by darkfeline** [magit/magit#5077] [Ideas]
Add support for non-populated submodules in `magit-list-submodules` (kind of like `magit-insert-modules-overview` section hook).

I init/deinit submodules a lot, so it's useful to get an overview of the status of all submodules.

**Comment by darkfeline** [comment]
Example implementation (kind of scuffed since we're matching on column names)  https://github.com/darkfeline/magit/commit/04919df5b92ad41f9eac8e2f353b05119596837f

**Comment by tarsius** [comment]
Implemented in 519674da457ab860b1454a98520d669d2e73035e.
